BAQE-1425 - Remove DummyProcessRuntime from session benchmarks

hellowdan opened the pull request:
On Drools benchmarks, the DummyProcessRuntime class is being used only on org.drools.benchmarks.session.sessionpool.DisposeSessionWithProcessRuntimeBenchmark and it is being affected by changes in the Interface InternalProcessRuntime (which is implemented by DummyProcessRuntime).

hellowdan commented:
Jenkins test again

tarilabs commented:
mmm... I think this is more a question for @mariofusco but I am afraid there was a good reason why the UnsupportedOpException

hellowdan commented:
@MarianMacik getProcessInstance() is called to force the creation of a process runtime and I removed the exception because of this (just like it was in the Dummy class which I removed from kie-benchmarks project to avoid compilation error there).

MarianMacik commented:
@hellowdan So the drools benchmarks actually call this method somewhere? I think this shouldn't happen unless you really use processes. As right now it is designed in a way that it will throw an exception.

baldimir commented:
@hellowdan as @tarilabs mentioned, this is a question for @mariofusco. I expect the exception was there for some reason. Please consult with Mario.

hellowdan commented:
Considering your feedbacks, this is not a good solution. Closing the PR, will think another approach. Thank you all.
